### PR TITLE
New content added to pages and images

### PR DESCRIPTION
**Pull request by WRRicht3r**

### CODE REVIEW

File: src/.DS_Store
Comment (bgodlin) [inline src/.DS_Store:1]: Can you please remove all `.DS_store` files?

File: src/adding-a-token-to-a-hashi-bridge.md
Comment (bgodlin) [inline src/adding-a-token-to-a-hashi-bridge.md:1]: > Adding a token to the SORA HASHI bridge involves several steps;
> 1. Registering a token within the SORA blockchain
> 2. Registering a SORA asset on the bridge
> 3. Registration of an ERC-20 token in Ethereum (and its mapping with a SORA asset)
> 4. Adding a token to a whitelist

Should we use anchor links here just like they are used in the current wiki?

Comment (bgodlin) [reply src/adding-a-token-to-a-hashi-bridge.md:1]: Line 12 still requires a link

File: src/consensus.md
Comment (bgodlin) [inline src/consensus.md:1]: Should we move this to `SORA economy` in the `Getting started` category?

**Review by bgodlin — CHANGES_REQUESTED** [review]
Can't build (https://vitepress.dev/guide/deploy) the project by running `docs:build`. The additional changes will have to be applied

File: src/adding-a-token-to-a-hashi-bridge.md
Comment (WRRicht3r) [reply src/adding-a-token-to-a-hashi-bridge.md:1]: Left blank for now since we don't have the article for the link yet

File: src/consensus.md
Comment (WRRicht3r) [reply src/consensus.md:1]: We could... At the moment I just populated the document

**Pull request merged**